### PR TITLE
Fix Slurm Runner

### PR DESCRIPTION
**Pull request by dashstander**
Fix SlurmRunner so that a comment can be added and parsed appropriately. @Quentin-Anthony

### CODE REVIEW

**Review by Quentin-Anthony — DISMISSED** [review]

**Comment by Quentin-Anthony** [comment]
This looks good. Keep in mind @dashstander that we'll need to rebase this once the merge to upstream DeepSpeed happens.

**Review by StellaAthena — DISMISSED** [review]

**Comment by StellaAthena** [comment]
@Quentin-Anthony I have no idea why your approval didn’t seem to work, but mine is working. You are listed in the `CODEOWNERS` file… can you double check that your username is spelled correctly? I dismissed your review because I thought maybe it was requiring two for some reason but that doesn’t seem to be the case.

**Review by Quentin-Anthony — APPROVED** [review]

**Review by StellaAthena — APPROVED** [review]

**Pull request merged**
